gpc-api-facade/src/main/java/uk/nhs/adaptors/pss/gpc/service/MigratePatientStructuredRecordService.java

```diff
@@ -0,0 +1,51 @@
+package uk.nhs.adaptors.pss.gpc.service;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.Parameters;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import lombok.AllArgsConstructor;
+import uk.nhs.adaptors.connector.dao.PatientMigrationRequestDao;
+import uk.nhs.adaptors.connector.model.RequestStatus;
+import uk.nhs.adaptors.pss.gpc.amqp.PssQueuePublisher;
+import uk.nhs.adaptors.pss.gpc.exception.FhirValidationException;
+
+@Component
+@AllArgsConstructor()
```

Comment:
Maybe @RequiredArgsConstructor(onConstructor = @__(@Autowired)) instead of @AllArgsConstructor?